tests/test_port.py

```diff
@@ -1,63 +1,69 @@
-import numpy as np
+from numpy.testing import assert_array_equal, assert_array_almost_equal
 import pandas as pd
 
 from sklearn_knn import Raw, Euclidean, GNN
 
+import pytest
 
-def get_check_dist_fn(project, method, ref_or_trg):
-    return f"./tests/data/{method}_{project}_{ref_or_trg}_distances_k5.csv"
 
+class Dataset:
```

Comment:
I guess we might want to re-organize if we end up adding additional data or want to make the test data available through the package, but I don't see anything wrong with it at the moment!

Reply:
Gave this a bit more thought.  If we plan to write examples around the Moscow Mountain / St. Joes data (like yaImpute does), maybe we do want to mimic the sklearn behavior of `load_dataset`.  Looking at that code, it looks like they are loading data into `sklearn.utils.Bunch` objects.  Do we want to follow this convention?  I also asked Andy Hudak if he was OK with us using that dataset in this package with attribution.  Haven't heard back yet.

Reply:
I like the idea of mimicking their `datasets` module with some `load_` functions. A special-purpose `Dataset` class seems more useful to me than their `Bunch` approach, since that would make it easier to keep attributes and types consistent across datasets. And if we can stick with their naming schemes (`data`, `target`, etc.) the difference should be pretty transparent to users. But let me know if you feel strongly that we should stick to `Bunch` for consistency.

Reply:
I'll let you drive this one.  I don't have any real knowledge of Bunch objects, so not wedded to that.

Reply:
Happy to! My understanding was that `Bunch` is just a glorified `dict` (similar to [Box](https://github.com/cdgriffith/Box) if you're familiar with that package).

Do you see this as part of this PR or a future one? My preference would be a future one, unless you think we need to get this set up to help with the testing data.

Reply:
Resolving this for now so we can track it separately in #3, since we've got preliminary testing data up and running without it.